IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -105,6 +105,9 @@ @implementation MSIDWebviewFactory
                                                              endURL:[NSURL URLWithString:configuration.endRedirectUrl]
                                                             webview:webview
                                                       customHeaders:configuration.customHeaders
+#if TARGET_OS_OSX
+                                                       customWindow:NSZeroRect
```

Comment:
can we have it as an OS X only property? Initializers with conditional parameters are quite difficult to read/maintain.

Reply:
Will make the change. Thanks!

Reply:
Not seeing the change for MSIDAADOAuthEmbeddedWebviewController. 
Regarding setting it in initializer... maybe you can pass a bag of properties? Like an object with all the parameters in the initializer. @antrix1989, any other ideas?

Reply:
Sorry missed the necessary change in that file. Will take the way of adding an object and make the change. Thanks for the advice!

Reply:
Addressed